assertions/src/test/java/io/camunda/zeebe/process/test/assertions/BpmnAssertTest.java

```diff
@@ -0,0 +1,100 @@
+package io.camunda.zeebe.process.test.assertions;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.mock;
+
+import io.camunda.zeebe.client.api.response.ActivatedJob;
+import io.camunda.zeebe.client.api.response.DeploymentEvent;
+import io.camunda.zeebe.client.api.response.ProcessInstanceEvent;
+import io.camunda.zeebe.client.api.response.ProcessInstanceResult;
+import io.camunda.zeebe.client.api.response.PublishMessageResponse;
+import io.camunda.zeebe.process.test.api.RecordStreamSource;
+import io.camunda.zeebe.process.test.inspections.model.InspectedProcessInstance;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BpmnAssertTest {
+
+  @BeforeEach
+  public void beforeEach() {
+    BpmnAssert.initRecordStream(mock(RecordStreamSource.class));
+  }
+
+  @AfterEach
+  public void afterEach() {
+    BpmnAssert.resetRecordStream();
+  }
+
+  @Test
+  public void testAssertThatProcessInstanceEventReturnsProcessInstanceAssert() {
```

Comment:
:wrench: We could follow the convention of JUnit 5 and remove the `public` modifier of the test method.

Additionally, we could think about using `@DisplayName("Should return ProcessInstanceAssert for ProcessInstanceEvent")` to have human-friendly test names.